components/brave_ads/core/internal/settings/settings.h

```diff
@@ -14,7 +14,10 @@ bool UserHasOptedInToBraveNewsAds();
 
 bool UserHasOptedInToNewTabPageAds();
 
+bool UserHasOptedInToSearchAds();
```

Comment:
nit: Can we please add after `UserHasOptedInToNotificationAds`, just because we sort elsewhere by name.